File: examples/user/logout/node/logAUserOut.js
const { UserService } = require("micro-js-client/user");

// Logout a user account
async function logAuserOut() {
  let userService = new UserService(process.env.MICRO_API_TOKEN);
  let rsp = await userService.logout({
    sessionId: "df91a612-5b24-4634-99ff-240220ab8f55",
  });
  console.log(rsp);
}

logAuserOut();

File: examples/user/logout/go/logAUserOut.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/user"
)

// Logout a user account
func LogAuserOut() {
	userService := user.NewUserService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := userService.Logout(&user.LogoutRequest{
		SessionId: "df91a612-5b24-4634-99ff-240220ab8f55",
	})
	fmt.Println(rsp, err)
}
